src/telliot_feeds/reporters/tellorflex_360.py

```diff
@@ -0,0 +1,223 @@
+        wait_period: int = 7,
+    ) -> None:
+
+        self.endpoint = endpoint
+        self.oracle = oracle
+        self.token = token
+        self.autopay = autopay
+        self.stake: Optional[int] = stake  # type: ignore
+        self.datafeed = datafeed
+        self.chain_id = chain_id
+        self.acct_addr = to_checksum_address(account.address)
+        self.last_submission_timestamp = 0
+        self.expected_profit = expected_profit
+        self.transaction_type = transaction_type
+        self.gas_limit = gas_limit
+        self.max_fee = max_fee
+        self.wait_period = wait_period
+        self.priority_fee = priority_fee
+        self.legacy_gas_price = legacy_gas_price
+        self.gas_price_speed = gas_price_speed
+        self.autopaytip = 0
+        self.staked_amount: Optional[float] = None
+        self.staker_info: Optional[StakerInfo] = None
+        self.allowed_stake_amount = 0
+
+        logger.info(f"Reporting with account: {self.acct_addr}")
+
+        self.account: ChainedAccount = account
+        assert self.acct_addr == to_checksum_address(self.account.address)
+
+    async def ensure_staked(self) -> Tuple[bool, ResponseStatus]:
+        """Compares stakeAmount and stakerInfo every loop to monitor changes to the stakeAmount or stakerInfo
+        and deposits stake if needed for continuous reporting
+
+        Return:
+        - (bool, ResponseStatus)
+        """
+        # get oracle required stake amount
+        stake, status = await self.oracle.read("getStakeAmount")
+
+        if (not status.ok) or (stake is None):
+            msg = "Unable to read current stake amount"
+            return False, error_status(msg, log=logger.info)
+
+        # get accounts current stake total
+        staker_info, status = await self.oracle.read(
+            "getStakerInfo",
+            _stakerAddress=self.acct_addr,
+        )
+        if (not status.ok) or (staker_info is None):
+            msg = "Unable to read reporters staker info"
+            return False, error_status(msg, log=logger.info)
+
+        if self.staker_info is None:
+            self.staker_info = StakerInfo(*staker_info)
+
+        if self.staker_info.stake_balance > staker_info[1]:
+            # update balance after depositing more stake
+            self.staker_info.last_report = staker_info[4]
+            self.staker_info.stake_balance = staker_info[1]
+            logger.info("your staked balance has decreased and account might be in dispute")
+        else:
+            self.staker_info.last_report = staker_info[4]
+
+        account_staked_bal = self.staker_info.stake_balance
+
+        if self.stake is None:
+            self.stake = stake
+
+        if stake > self.stake:
+            self.stake = stake
+            msg = "Stake amount has increased due to TRB price change."
+            logger.info(msg)
+
+        if stake < self.stake:
+            self.stake = stake
+            msg = "Stake amount has decreased,"
```

Comment:
Don't need a comma at the end of `msg`. Also, don't need to instantiate a string variable here if you're only using it once.